src/main/java/laststep/domain/Cars.java

```diff
@@ -0,0 +1,69 @@
+package laststep.domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import laststep.service.MoveStrategy;
+import laststep.service.RandomMoveStrategy;
+
+public class Cars {
+    private final MoveStrategy strategy;
+    private final List<Car> cars;
+
+    public Cars(List<Car> cars) {
+        validate(cars);
+        this.strategy = new RandomMoveStrategy();
+        this.cars = cars;
+    }
+
+    public void race() {
+        for (Car car : cars) {
+            car.move(strategy);
+        }
+    }
+
+    public Cars findWinners() {
+        Car winner = findWinnerPosition();
+        return new Cars(cars.stream()
+                .filter(car -> car.isWinner(winner))
+                .collect(Collectors.toList()));
```

Comment:
여기서도 toUnmodifiableList() 를 적용해 보시면 어떨까요?
```suggestion
                .collect(Collectors.toUnmodifiableList()));
```